CHANGELOG.md

```diff
@@ -17,7 +17,7 @@ You should check that your accordions still display as expected if you:
 
 You do not need to make changes to the content that accordions hide or show - this release's changes do not affect it.
 
-These changes were introduced in [pull request #2257: Update accordion design](https://github.com/alphagov/govuk-frontend/pull/2257).
+These changes were introduced in [pull request #2257: Update accordion design](https://github.com/alphagov/govuk-frontend/pull/2257). Thanks to [Mia Allers](mia-allers-gds), [Owen Jones](https://github.com/owenatgov), [Nikin Nagewadia](https://github.com/nnagewad) and [Chris Yoong](https://github.com/chris-gds) for their contributions.
```

Comment:
Is `mia-allers-gds` a GitHub username?

```suggestion
These changes were introduced in [pull request #2257: Update accordion design](https://github.com/alphagov/govuk-frontend/pull/2257). Thanks to [Mia Allers](https://github.com/mia-allers-gds), [Owen Jones](https://github.com/owenatgov), [Nikin Nagewadia](https://github.com/nnagewad) and [Chris Yoong](https://github.com/chris-gds) for their contributions.
```

Reply:
Yeah, I asked Mia and that's her current GitHub username.

Reply:
Cool, I think the `https://github.com/` part is missing from the link, currently it isn't working.

Reply:
D'oh. Sorry, I must be brain-fogged. Thanks for fixing!